content/docs/plugins.md

```diff
@@ -85,8 +85,54 @@ for any technical questions/answers and the announcement your new plugin.
 
 A lexer plugin is a kind of extension compared to a normal plugin,   
 i.e. in addition to all the functions already mentioned, additional ones have to be exported and also an XML file defining the styles has to be generated.  
-  
-Notepad++ (Npp) currently supports only the [ILexer4](https://www.scintilla.org/ScintillaDoc.html#LexerObjects) interface.  
+
+#### Notepad++ v8.4 and newer
+
+Notepad++ has transitioned to the [ILexer5](https://www.scintilla.org/LexillaDoc.html) interface as of Notepad++ v8.4.
+A lexer plugin needs to define all methods of this interface to ensure a smooth interaction.  
+
+The lexer functions to be exported by the plugin are
+- GetLexerCount  
+- GetLexerName  
+- CreateLexer
+- LexerNameFromID
+
+The [`SCI_SETILEXER`](https://www.scintilla.org/ScintillaDoc.html#SCI_SETILEXER) message is used to set the current lexer to your defined ILexer5 object.
+
+The [Notepad++ Community Forum](https://community.notepad-plus-plus.org/) has a discussion on how to "[Make External Lexer Plugin work with v8.4](https://community.notepad-plus-plus.org/topic/22866/make-external-lexer-plugin-work-with-v8-4)".
+
+If you are starting with an ILexer4 lexer defined as below, to convert to ILexer5, it should work to simply add a `CreateLexer` function that returns the pointer to the ILexer5 interface.  In the future, Notepad++ may add support for the GetLibraryPropertyNames, SetLibraryProperty and GetNameSpace functions; if lexer-plugin developers find a good reason and want that support, they should [make an official feature request](https://github.com/notepad-plus-plus/notepad-plus-plus/issues).
```

Comment:
The external lexers done with an ILexer4 interface (with Scintilla 4.x headers), need to update their old Scintilla headers (Scintilla v4.x) to the new ones (Scintilla v5.x), and in additional add `CreateLexer`.
("PapyrusPlugin" has had upgrade to Scintilla5 before I adding `CreateLexer` in the plugin to make it work - I might be not clear on my commit comment on this part).

I suggest reword the above sentence to the following one:
*If you are starting with an ILexer4 lexer defined as below, to convert to ILexer5, you should update Scintilla headers to Scintilla5, and add a `CreateLexer` export function that returns the pointer to the ILexer5 interface.  In the future...*